hexapod:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  right_front_1: 
    type: effort_controllers/JointPositionController
    joint: rf1_joint
    pid : {p: 10.0 , i: 1.0 , d: 0.0}

  right_front_2: 
    type: effort_controllers/JointPositionController
    joint: rf2_joint
    pid : {p: 10.0 , i: 1.0 , d: 0.0}

  right_front_3: 
    type: effort_controllers/JointPositionController
    joint: rf3_joint
    pid : {p: 10.0 , i: 1.0 , d: 0.0}

  right_middle_1: 
    type: effort_controllers/JointPositionController
    joint: rm1_joint
    pid : {p: 10.0 , i: 1.0 , d: 0.0}
  
  right_middle_2: 
    type: effort_controllers/JointPositionController
    joint: rm2_joint
    pid : {p: 10.0 , i: 1.0 , d: 0.0}

  right_middle_3: 
    type: effort_controllers/JointPositionController
    joint: rm3_joint
    pid : {p: 10.0 , i: 1.0 , d: 0.0}
  
  right_back_1: 
    type: effort_controllers/JointPositionController
    joint: rb1_joint
    pid : {p: 10.0 , i: 1.0 , d: 0.0}

  right_back_2: 
    type: effort_controllers/JointPositionController
    joint: rb2_joint
    pid : {p: 10.0 , i: 1.0 , d: 0.0}

  right_back_3: 
    type: effort_controllers/JointPositionController
    joint: rb3_joint
    pid : {p: 10.0 , i: 1.0 , d: 0.0}


  left_front_1: 
    type: effort_controllers/JointPositionController
    joint: lf1_joint
    pid : {p: 10.0 , i: 1.0 , d: 0.0}

  left_front_2: 
    type: effort_controllers/JointPositionController
    joint: lf2_joint
    pid : {p: 10.0 , i: 1.0 , d: 0.0}

  left_front_3: 
    type: effort_controllers/JointPositionController
    joint: lf3_joint
    pid : {p: 10.0 , i: 1.0 , d: 0.0}

  left_middle_1: 
    type: effort_controllers/JointPositionController
    joint: lm1_joint
    pid : {p: 10.0 , i: 1.0 , d: 0.0}
  
  left_middle_2: 
    type: effort_controllers/JointPositionController
    joint: lm2_joint
    pid : {p: 10.0 , i: 1.0 , d: 0.0}

  left_middle_3: 
    type: effort_controllers/JointPositionController
    joint: lm3_joint
    pid : {p: 10.0 , i: 1.0 , d: 0.0}
  
  left_back_1: 
    type: effort_controllers/JointPositionController
    joint: lb1_joint
    pid : {p: 10.0 , i: 1.0 , d: 0.0}

  left_back_2: 
    type: effort_controllers/JointPositionController
    joint: lb2_joint
    pid : {p: 10.0 , i: 1.0 , d: 0.0}

  left_back_3: 
    type: effort_controllers/JointPositionController
    joint: lb3_joint
    pid : {p: 10.0 , i: 1.0 , d: 0.0}